real_time/rbsm_config.txt

```diff
@@ -2,7 +2,7 @@
 
 // RBSM Headers
 
-RBSM_MID_ENC_TICKS_LAST, 0
```

Comment:
Was this commented out for debugging purposes or are we planning on removing it?